src/components/DocumentSectionBox/index.tsx

```diff
@@ -3,26 +3,26 @@ import { Link } from "react-router-dom";
 import { TablerIcon } from "../../utils/types";
 
 export default function DocumentSectionBox({
-  section,
+  title,
+  icon: IconComponent,
+  documents,
 }: {
```

Comment:
erm do `document` instead of `{
  section,
  title,
  icon: IconComponent,
  documents
}` :))

Reply:
ur choice tho

Reply:
oh bruh i thought u wanted to split into multiple props

Reply:
whatever i've been doing this for other react projects